src/arrayMethodPop.js

```diff
@@ -5,7 +5,15 @@
  */
 function applyCustomPop() {
   [].__proto__.pop2 = function() {
-    // write code here
+    const items = this;
+    let lastElement;
```

Comment:
це спрацює також array[-1] поверне undefined
```suggestion
    let lastElement = items[items.length - 1];
```